src/journal/cat.c

```diff
@@ -122,10 +122,7 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
 
                 case ARG_NAMESPACE:
-                        if (isempty(optarg))
-                                arg_namespace = NULL;
```

Comment:
If you do this, please also update `arg_identifier` above

Reply:
Done.